### PR TITLE
ci: Enable markdown linting and spell checking

### PR DESCRIPTION
**Pull request by agneszitte**

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

**Review by kazo0 — APPROVED** [review]

File: doc/material-migration.md [hunk 1]
```diff
@@ -81,28 +83,30 @@ MaterialNavigationTabBarItemActiveIndicatorWidth|NavigationTabBarItemActiveIndic
...
- There are two available paths once you have updated to the new Material Toolkit v2 package: 
+ There are two available paths once you have updated to the new Material Toolkit v2 package:
 
 - Continue to keep using the v1 styles; or,
 - Migrate app to reference the new v2 styles
 
-
-## Continue Using v1 Styles
+### Continue Using v1 Styles
+>
```
Comment (Xiaoy312) [inline doc/material-migration.md:100]: empty line after header?
```suggestion

```

File: build/workflow/cspell.json
Comment (Xiaoy312) [inline build/workflow/cspell.json:1]: a lot of these words arnt even used here
did you copy pasted them from a template or elsewhere?

**Review by Xiaoy312 — APPROVED** [review]

File: build/workflow/cspell.json
Comment (agneszitte) [reply build/workflow/cspell.json:1]: It is the same values from the cspell.json @jeromelaban used for the Uno.Themes repo when enabling it.
@jeromelaban should we use only what is needed or we can go with these general values when enabling it on all repos the first time ?

File: doc/material-migration.md (hunk 1 above)
Comment (agneszitte) [reply doc/material-migration.md]: Fixed, thanks for noticing!

File: build/workflow/cspell.json
Comment (jeromelaban) [reply build/workflow/cspell.json:1]: Let's keep the list, it's not too much of a problem to have non-used words, we excluded those once, we'll do it again anyways :)

Comment (agneszitte) [reply build/workflow/cspell.json:1]: Ok thanks for the confirmation @jeromelaban !

**Pull request merged**